windows/jitsi-meet/index.html

```diff
@@ -18,6 +18,7 @@
         </style>
     </head>
     <body>
+      <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
is this needed by the main window?  i would've thought we were already pulling in adapter in lib-jitsi-meet